base/base-docset-owner-guide.md

```diff
@@ -0,0 +1,44 @@
+# {Doctype} customization guide
+
+This guide describes a {doctype} document's:
+* Unique characteristics,
+* When to include the {doctype} dotype in a project's documentation set, and
+* How [{doctype} template-set templates](../{doctype}) should be customized by a docset-owner to address a project's needs.
+
+
+**Version:** {This document's version, in MAJOR.MINOR.PATCH format}
+
+**Last updated:** {Date in the format: *Month DD, _YYYY* or *Month, YYYY*}
+
+## base-template customization
+
+TBD: Explain the unique characteristics of this section.
+
+## base-example
+
+The {doctype}-example.md document should describe the fictional [chronologue](https://github.com/thegooddocsproject/chronologue/) project.
+
+We've chosen to use a common example project to reduce cognitive load on the reader, and to help build a common theme between templates.
+
+A fictional project was chosen over a real example because:
+* The fictional project's characteristics can remain static over time, ensuring examples don't need to be updated.
+* Characteristics can be invented to illustrate a specific feature.
+* We avoid having the select a "favorite" exemplar project, and upsetting non-selected projects. 
+* Examples and features can be a little whimsical, adding a little entertainment to the reading. Note, any whimsical additions should not come at the expense of legibility for the wide target audience.
+
+The `chronologue time machine` concept was chosen because:
```

Comment:
**Nitpick (non-blocking):** It's not a time machine itself, it's an app that synchronizes your clocks to the time zone you're traveling to.

Reply:
Hmm, can we make sure the fake project can be explained very concisely, in a couple of words, such as I'm attempting to do here. Maybe that should be a criteria for the fake project.